DownFall/TileStrategy.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  TileStrategy.swift
+//  DownFall
+//
+//  Created by William Katz on 1/8/19.
+//  Copyright © 2019 William Katz LLC. All rights reserved.
+//
+
+import GameplayKit
+
+protocol TileStrategy {
+    static var randomSource: GKLinearCongruentialRandomSource { get }
+    static func tiles(for board: Board, difficulty: Difficulty) -> [TileType]
+}
```

Comment:
let's think about adding more functionality that creates a board from only a difficulty and board size